test/e2e-test/flareact-error.test.ts

```diff
@@ -0,0 +1,104 @@
+import util = require('node:util');
+// eslint-disable-next-line @typescript-eslint/no-var-requires, no-undef
+const execFile = util.promisify(require('node:child_process').exec);
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const copy = require('recursive-copy');
+
+import AWS = require('aws-sdk');
+
+import { expect } from 'chai';
+
+describe('Create Flareact application with error', () => {
+    let templatePath: string;
+    let template: string;
+    let realPath: string;
+    let localOutput: string;
+    const bucketParams = {
+        Bucket: "azion-test"
+    }
+
+    const s3 = new AWS.S3 ({
+        accessKeyId: "123456",
+        secretAccessKey: "123456",
+        signatureVersion: "v4",
+        s3ForcePathStyle: true,
+        endpoint: "http://localhost:4566"
+    });
+
+    before(async () => {
+
+        await s3.createBucket(bucketParams).promise();
+    });
+
+    describe('Build the Flareact project without "azion.json" file and expect it fail', () => {
+        before(async () => {
+            // Creates temporary local template repository
+            templatePath = fs.mkdtempSync(path.join(os.tmpdir(), 'flareact-test'));
+            realPath = fs.realpathSync(templatePath);
+            const templateName = 'flareact-template';
+            template = path.join(realPath, templateName);
+            localOutput = process.cwd();
+        });
+
+        after(() => {
+            process.chdir(localOutput);
+            if (templatePath) {
+                fs.rmSync(templatePath, { recursive: true });
+            }
+        });
+
+        it('init template flareact"', async () => {
+            const { stdout} = await execFile(`azion-framework-adapter init ${template} https://github.com/flareact/flareact-template`);
+            expect(stdout).to.be.equal('Completed.\n');
+        });
+
+        it('Install packages', async () => {
+            process.chdir(template);
+            await execFile('npm install', {});
+            const packageLock = fs.existsSync(path.join(template,'package-lock.json'));
+            expect(packageLock).to.be.true;
+        });
+
+        it('Build the Flareact project without "azion.json" file and expect it fail', async () => {
+            const expectOnError =  "Couldn't read file 'azion.json' at the project's root directory. Because ENOENT: no such file or directory, open 'azion.json'\n"
+
+            function run(cmd: string) {
+                return new Promise((resolve, _reject) => {
```

Comment:
As you did "promesify" the `child_process.exec`, you probably don't need to use it with a "callback". It should return a promise.